telegram/commands/gemini/gemini.go

```diff
@@ -0,0 +1,28 @@
+package gemini
+
+import (
+	"activity-tracker/gemini"
+	"activity-tracker/shared"
+	"errors"
+	"fmt"
+)
+
+var (
+	errNoPrompt = errors.New("En serio metiste un prompt vacío bobo hijueputa? Haz algo bien sub-humano 🙄")
+	forceText   = "responde solo texto"
```

Comment:
just out of consistent, add "," at the beginning of the string value

Reply:
solved